.pipelines/templates/template-run-rp-and-e2e.yml

```diff
@@ -2,6 +2,7 @@ parameters:
   workingDirectory: ""
 steps:
   - script: |
+      #!/bin/bash -e
```

Comment:
`set -e`

Reply:
(we should be aiming for this to be set across the board, but let's do that separately)